src/classiceditor.js

```diff
@@ -192,12 +194,14 @@ export default class ClassicEditor extends Editor {
 					.then( () => editor.ui.init() )
 					.then( () => {
 						if ( isElement( sourceElementOrData ) ) {
-							editor._elementReplacer.replace( sourceElementOrData, editor.element );
+							editor._elementReplacer.replace( sourceElementOrData, editor.ui.element );
 						}
 
+						editor.ui.ready();
```

Comment:
It would be great to move `_elementReplacer` to `editor.ui`. Also, it could be `editor.ui.init` what fire `ready` event.